src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -465,8 +478,23 @@ private function loadCacheTagging(ContainerBuilder $container, XmlFileLoader $lo
         }
 
         $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', true);
-        $container->setParameter($this->getAlias().'.tag_handler.response_header', $config['response_header']);
+
         $container->setParameter($this->getAlias().'.tag_handler.strict', $config['strict']);
+
+        // If no response_header parameter has been already been set, use the one from the config
+        if (!$container->hasParameter($this->getAlias().'.tag_handler.response_header')) {
+            if (isset($config['response_header'])) {
+                $container->setParameter($this->getAlias().'.tag_handler.response_header', $config['response_header']);
+            } else {
+                $container->setParameter($this->getAlias().'.tag_handler.response_header', 'X-Cache-Tags');
```

Comment:
this if/else could be shortened with PHP 7.1 syntax to `$container->setParameter($this->getAlias().'.tag_handler.response_header', $config['response_header'] ?? Varnish::DEFAULT_HTTP_HEADER_CACHE_TAGS);` (please keep using the constant for this header)